internal/pkg/utils/errors/errors.go

```diff
@@ -0,0 +1,35 @@
+// Package errors extends standard error handling with MultiError and error's stack trace.
```

Comment:
Ak bude nieco nejasne, tak pridam dalsie komentare.
Doteraz sme pouzivali v kode rozne balicky na spracovanie chyb, teraz staci tento jeden a obsahuje vsetko.
Jednoduchsie tak vieme nieco ohladom chyb upravit.

Kazda chyba obsahuje `stack trace`, info kde vznikla, co moze pomoct pri debugovani (standardne sa to nezobrazuje, ale da sa to vypisat s `FormatWithDebug`).